### ISSUE TITLE
Unable to log in to cluster when using username and password

### ISSUE THREAD

**Issue by dipak-pawar** [fabric8io/kubernetes-client#1055]
```java
public class KubernetesClientTest {

    @Test
    public void k8s_client_auth_token() {
        final DefaultKubernetesClient kubernetesClient = new DefaultKubernetesClient(new ConfigBuilder()
            .withMasterUrl("https://api.starter-us-east-2.openshift.com")
                .withOauthToken("token")
            .build());

        final OpenShiftClient openShiftClient = kubernetesClient.adapt(OpenShiftClient.class);

        openShiftClient.projects().list();
    }

    @Test
    public void k8s_client_username_password() {
        final DefaultKubernetesClient kubernetesClient = new DefaultKubernetesClient(new ConfigBuilder()
            .withMasterUrl("https://api.starter-us-east-2.openshift.com")
                .withUsername("username")
                .withPassword("password")
                .withNamespace("namespace")
            .build());

        final OpenShiftClient openShiftClient = kubernetesClient.adapt(OpenShiftClient.class);

        openShiftClient.projects().list();
    }
}
```

If you try to run above test, then `k8s_client_username_password` will fail by saying unathourized, whereas `k8s_client_auth_token` tests will pass successfully.

However both tests should pass

**Comment by piyush1594** [comment]
closing this in #1056

**Comment by piyush1594** [comment]
Hey, @dipak-pawar fix for this is available in  Kubernetes Client 3.2.0
